app/ui-react/syndesis/src/modules/connections/pages/create/ConfigurationPage.tsx

```diff
@@ -30,6 +30,46 @@ export interface IConfigurationPageRouteState {
   connector: Connector;
 }
 
+interface IOAuthFlowProps {
+  connector: Connector;
```

Comment:
If we want to nitpick about this, I'd say that we could avoid passing the whole connector object and ask for the id and name instead, and instead of history we could ask for the callback to fire on a successful connect. And then this would be ready to be used in the creation flow too :)